resource-managers/kubernetes/integration-tests/src/test/scala/org/apache/spark/deploy/k8s/integrationtest/DepsTestsSuite.scala

```diff
@@ -58,7 +59,7 @@ private[spark] trait DepsTestsSuite { k8sSuite: KubernetesSuite =>
     ).toArray
 
     val resources = Map(
-      "cpu" -> new Quantity("1"),
+      "cpu" -> new Quantity(sys.props.get(CONFIG_MINIO_REQUEST_CORES).getOrElse("1")),
```

Comment:
BTW, do we need a parameter for this? This is not a Spark configuration. I guess we can have a simply small value until it works.

Reply:
Yep, I also considered this before. I think 0.25 (250m) could be a reasonable smaller value in here. Because I also see this experience value in [link]
(https://docs.gitlab.com/charts/charts/minio/#installation-command-line-options).